clients/drcachesim/tracer/instru_offline.cpp

```diff
@@ -323,6 +333,7 @@ offline_instru_t::append_thread_header(byte *buf_ptr, thread_id_t tid,
     new_buf += sizeof(*entry);
     new_buf += append_tid(new_buf, tid);
     new_buf += append_pid(new_buf, dr_get_process_id());
+    new_buf += append_target_cache_line_size(new_buf);
```

Comment:
(Hmm, there's no thread-shared single place to place this: putting in just one file makes it harder to parallelize, so putting in *every* thread file is the thing to do I guess.)

Reply:
Acknowledged.